edgecompute/examples/authentication/jwt/main.js

```diff
@@ -0,0 +1,62 @@
+import { crypto,pem2ab } from 'crypto';
+import { TextEncoder, base64url } from 'encoding';
+
+async function verify_es256(jwt, key) {
+    // Split the signature from the header/claims
+    const lastDot = jwt.lastIndexOf('.');
+    if (lastDot < 0) {
+        throw new Error("Missing dot");
+    }
+    const headerAndClaims = jwt.substring(0, lastDot);
+    const signature = jwt.substring(lastDot + 1);
+ 
+    return crypto.subtle.verify({
+            name: "ECDSA",
+            hash: "SHA-256"
+        },
+        key,
+        base64url.decode(signature),
+        new TextEncoder().encode(headerAndClaims)
+    ).then(verified => {
+        if (verified) {
+            // Success! Extract the header and claims.
+            const [header, claims] = headerAndClaims.split('.');
+            return [
+                JSON.parse(base64url.decode(header, "String")),
+                JSON.parse(base64url.decode(claims, "String"))
+            ];
+        } else {
+            throw new Error("signature did not match")
```

Comment:
Does this get caught anywhere?

Reply:
Yes, this is the sample code on the techdocs below.
https://techdocs.akamai.com/edgeworkers/docs/crypto#example-2---verify-a-jwt-with-hmac